Packs/IronDefense/IncidentFields/incidentfield-IronDefense_Alert_ID.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
You can remove this incident field and use the "incidentfield-IronDefense_Alert_IDs.json" for this incident type too

Reply:
Right I believe you mentioned this on our call the other day. We have both because we use Alert ID when there can only be exactly one ID and Alert IDs when there can be many.
Would removing one of these just be to reduce the number of fields by one? I would think it's better to have the field name display the name we want.